server/fleet/app.go

```diff
@@ -1205,5 +1230,5 @@ func (v *Version) AuthzType() string {
 type WindowsSettings struct {
 	// NOTE: These are only present here for informational purposes.
 	// (The source of truth for profiles is in MySQL.)
-	CustomSettings optjson.Slice[string] `json:"custom_settings"`
+	CustomSettings optjson.Slice[MDMProfileSpec] `json:"custom_settings"`
```

Comment:
Mostly curious: why is this field an `optjson.Slice` whereas the equivalent MacosSettings field is a
regular slice?

Reply:
historic reasons, we did a bunch of different stuff to support optional values in these JSON structs until Martin came up with the `optjson` package. It would be great to go back and refactor the macOS implementation.

Reply:
Ah gotcha, makes sense!